.codecov.yml

```diff
@@ -11,11 +11,10 @@ coverage:
   status:
     project:
       default:
-        against: parent
-        target: auto
         threshold: 0.5%
-        branches:
-          - master
+    patch:
+      default:
+        target: 80%
```

Comment:
+1; This implies the diff coverage should be 80 % (auto implies PR base branch coverage is used )  and overall coverage can be reduced by 0.5%